torch/csrc/jit/backends/backend_init.cpp

```diff
@@ -0,0 +1,126 @@
+#include <torch/csrc/jit/backends/backend_init.h>
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/runtime/custom_operator.h>
+
+namespace torch {
+namespace jit {
+void initJitBackendBindings(PyObject* module) {
+  for (auto& backend : getAllBackends()) {
+    // Register custom ops for backend.compile and backend.execute so that the
+    // LoweredModule below can call them.
+    std::string compile_method_schema =
+        backend.name + "::compile(Any[] a) -> Any[]";
+    std::string execute_method_schema =
+        backend.name + "::execute(Any[] a) -> Any[]";
+
+    torch::jit::RegisterOperators reg({
```

Comment:
These registrations won't need to exist when the object is bound in as a class rather than as builtins. I think that is the right solution to this problem. If we want to split this apart, I'd register these without schema for now to get around the Operation definition. 

I worry however, that landing this and then landing a future potential update to make it classes might cause a BC problem. We need to be clear that there is no BC for this version of to_backend until the change to classes happens.